### PR TITLE
Added Abhishek's restaurant recommendation

### LINKED ISSUE #7: Create a file for Pune, India recommendations
# Pune, India

- [ ] Create branch
- [ ] Add commit
- [ ] Open a pull request
- [ ] Have a discussion
- [ ] Seek approval
- [ ] Merge pull request

### PR DESCRIPTION
**Pull request by abhikabra8811**
closes #7

### CODE REVIEW

File: abhikabra8811-Pune.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Pune, India
+
+### Places to Eat
+- Vrushali, F.C. Road : Best South Indian food
```
Comment (aaro7487) [inline abhikabra8811-Pune.md]: I've heard about this place but never been there. What would you suggest to try from here?

**Comment by aaro7487** [comment]
:smiley:

File: abhikabra8811-Pune.md (hunk 1 above)
Comment (abhikabra8811) [reply abhikabra8811-Pune.md]: Dosa, Uttappa, SPDP (Sev Potato Dahi Puri).

**Review by aaro7487 — APPROVED** [review]
:cool:  :+1:

**Pull request merged**